lib/mousestats/index.js

```diff
@@ -43,19 +43,48 @@ MouseStats.prototype.initialize = function(page){
  */
 
 MouseStats.prototype.loaded = function(){
-  return is.array(window.MouseStatsVisitorPlaybacks);
+  return !! (window.MouseStatsSharedControl && window.MouseStats_Commands);
 };
 
 /**
  * Identify.
  *
- * http://www.mousestats.com/docs/wiki/7/how-to-add-custom-data-to-visitor-playbacks
+ * http://www.mousestats.com/docs/wiki/27/identify-a-user
  *
  * @param {Identify} identify
  */
 
 MouseStats.prototype.identify = function(identify){
-  each(identify.traits(), function(key, value){
-    window.MouseStatsVisitorPlaybacks.customVariable(key, value);
+  var id = identify.userId() || identify.anonymousId();
+
+  if (typeof id !== 'string') id = '' + id;
```

Comment:
MouseStats_Commands.push(["tag", "key", "value"]) only accepts strings. It would be safe to pass a string instead of an unknown type.